### PR TITLE
added docker compose for mongo

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+db:
+  image: mongo:2.6
```
Comment (noirbizarre) [inline docker-compose.yml]: Why not latest available version ?

Comment (vinyll) [inline docker-compose.yml]: Just because the doc mentioned about `2.6+`. Would you up for `latest` then?

Comment (noirbizarre) [inline docker-compose.yml]: I was working with 3.2 without issues.
We might raise the 2.6+ to 3.2+ or let 2.6+ until someone signal it's not working anymore.

**Comment by vinyll** [comment]
Moved to _3.2_

**Comment by noirbizarre** [comment]
r+

**Pull request merged**